ui/desktop/app/routes/scopes/scope/projects/sessions.js

```diff
@@ -93,6 +93,12 @@ export default class ScopesScopeProjectsSessionsRoute extends Route {
   async cancelSession(session) {
     await session.cancelSession();
     await this.ipc.invoke('stop', { session_id: session.id });
+    if (
```

Comment:
We could make this action take in an argument and pass the route name from the template.

Reply:
Since we only redirect in one specific case, I feel this is the simplest approach. This is a good idea if we need to make this more complex though!